core/src/services/dropbox/backend.rs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use async_trait::async_trait;
+use http::StatusCode;
+
+use std::{fmt::Debug, sync::Arc};
+
+use crate::{
+    ops::{OpDelete, OpRead, OpWrite},
+    raw::{
+        parse_into_metadata, Accessor, AccessorInfo, HttpClient, IncomingAsyncBody, RpDelete,
+        RpRead, RpWrite,
+    },
+    types::Result,
+    Capability, Error, ErrorKind,
+};
+use super::{core::DropboxCore};
+
+#[derive(Clone, Debug)]
+pub struct DropboxBackend {
+    core: Arc<DropboxCore>,
+}
+
+
+impl DropboxBackend {
+    pub(crate) fn new(access_token: String, http_client: HttpClient) -> Self {
+        DropboxBackend {
+            core: Arc::new(DropboxCore {
+                token: access_token,
+                client: http_client,
+            }),
+        }
+    }
+}
+
+#[async_trait]
+impl Accessor for DropboxBackend {
+    type Reader = IncomingAsyncBody;
+    type BlockingReader = ();
+    type Writer = ();
+    type BlockingWriter = ();
+    type Pager = ();
+    type BlockingPager = ();
+    // async fn delete(&self, path: &str, _: OpDelete) -> Result<RpDelete> {
```

Comment:
This change seems not meet the title: `Support create/read/rename/copy/delete for Dropbox`, we should implement `Accessor` APIs.